onnxruntime/core/mlas/inc/mlas.h

```diff
@@ -89,6 +91,57 @@ typedef enum { CblasNonUnit=131, CblasUnit=132 } CBLAS_DIAG;
 typedef enum { CblasLeft=141, CblasRight=142} CBLAS_SIDE;
 #endif
 
+#ifdef MLAS_STANDALONE_LIB
```

Comment:
We can just assume MLAS is always standalone, and we don't need to keep the #else part.

Reply:
Removed.